src/Protocol/Diaspora.php

```diff
@@ -2128,7 +2175,32 @@ private static function receiveMessage($importer, $data)
 	 */
 	private static function receiveParticipation($importer, $data)
 	{
-		// I'm not sure if we can fully support this message type
```

Comment:
Just a side note, but in the future can you sign your comments in the first person or use a more impersonal voice? Every time I encounter such a comment, I feel like I'm working on your personal project and not a multi-contributor project.

Reply:
You are right, sorry. I'm not the type of impersonal voice - that's too impersonal. ;-) But I will keep an eye on it.

Reply:
You can keep the first person voice and just sign `— @annando` or something like that.

Reply:
This is like I'm doing at my job.